include/af/graphics.h

```diff
@@ -376,6 +390,27 @@ AFAPI af_err af_draw_plot(const af_window wind, const af_array X, const af_array
    C Interface wrapper for drawing an array as a plot
 
    \param[in]   wind is the window handle
+   \param[in]   X is an \ref af_array with the x-axis data points
+   \param[in]   Y is an \ref af_array with the y-axis data points
+   \param[in]   props is structure \ref af_cell that has the properties that are used
+   \param[in] marker is an \ref markerType enum specifying which marker to use in the scatter plot
+   for the current rendering.
+
+   \return     \ref AF_SUCCESS if rendering is successful, otherwise an appropriate error code
+   is returned.
+
+   \note \p X and \p Y should be vectors.
+
+   \ingroup gfx_func_draw
+*/
+AFAPI af_err af_draw_scatter(const af_window wind, const af_array X, const af_array Y, const af_cell* const props, const af_marker_type marker);
```

Comment:
@syurkevi Can you make the `props` variable the last argument in the function, that is more consistent with all the other `af_draw_*` functions.